installer/conf/syslog-ng-lad.conf

```diff
@@ -0,0 +1,31 @@
+# OMS Syslog collection for workspace LAD (SYSLOG_PORT=%SYSLOG_PORT%) (LAD_oms marker for unconfig)
+# To be added for each facility/min-level: log { source(src); filter(f_LAD_oms_f_<facility>); filter(f_LAD_oms_ml_<min_level>); destination(d_LAD_oms); };
+destination d_LAD_oms { udp("127.0.0.1" port(%SYSLOG_PORT%)); };
```

Comment:
To follow [convention](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/conf/syslog-ng.conf), the destination line should be after the filter lines.

With this, why don't you have a log line?